Rubberduck.Parsing/ComReflection/ComVariant.cs

```diff
@@ -53,11 +55,12 @@ private struct Variant
 
         public VarEnum VariantType { get; }
         public object Value { get; }
+        public string TypeName => TypeNames.TryGetValue(VariantType, out var typeName) ? typeName : "Object";
 
         public ComVariant(IntPtr variant)
         {
             Value = Marshal.GetObjectForNativeVariant(variant);
-            var members = (Variant)Marshal.PtrToStructure(variant, typeof(Variant));
+            var members = Marshal.PtrToStructure<Variant>(variant);
```

Comment:
Leaking a pointer here?

Reply:
I was looking at that last night thinking the same same thing. IIR, my rationale for doing that was that the Variant structures in the TypeInfo aren't really in working memory and they'll always be either intrinsic types or contain null data pointers. But... the only reason that's in there is to get default values for things like constants and parameters, there probably isn't a reason to hang onto the object.